index.html

```diff
@@ -352,183 +324,151 @@ <h2 class="project-title">
               <li>HTML</li>
               <li>CSS</li>
               <li>Javascript</li>
             </ul>
-            </div>
-            <button class="card-button" type="button">See Project</button>
           </div>
+          <button class="card-button" type="button">See Project</button>
         </div>
       </div>
-    </section>
-    
-    <footer>
+    </div>
+  </section>
+
+  <footer>
     <div id="wrapper-for-about-self-and-contact-form">
-    <section>
-      <div class="about-myself-and-skill-list-section">
-        <div class="about-myself-section">
-          <div class="container-for-about-section">
-            <div class="about-section-item">
-              <h1 class="intro">About Myself</h1>
-            </div>
-            <div class="about-section-item">
-              <p class="profile">
-                Hello I’m a software developer! I can help you build a product ,
-                feature or website. Look through some of my work and experience!
-                If you like what you see and have a project you need coded,
-                don’t hesitate to contact me.
-              </p>
-            </div>
-            <div class="about-section-item links">
-              <a href="http://" class="lets-connect">LET’S CONNECT</a>
-              <ul class="social-media-icons">
-                <li class="individual-icons">
-                  <a href="#"
-                    ><img class="git" src="./assets/git.png" alt="git"
-                  /></a>
-                </li>
-                <li class="individual-icons">
-                  <a href="#">
-                    <img
-                      class="link"
-                      src="./assets//Linkedin icon.png"
-                      alt="linkedin"
-                    />
-                  </a>
-                </li>
-                <li class="individual-icons">
-                  <a href="#"
-                    ><img
-                      class="twohandicon"
-                      src="./assets/twohandicon.png"
-                      alt="my pub"
-                  /></a>
-                </li>
-                <li class="individual-icons">
-                  <a href="#"
-                    ><img
-                      class="twitter"
-                      src="./assets//twitter.png"
-                      alt="twitter"
-                  /></a>
-                </li>
-                <li class="individual-icons">
-                  <a href="#"
-                    ><img
-                      class="socialIcon"
-                      src="./assets/socialIcon.png"
-                      alt="social"
-                  /></a>
-                </li>
-              </ul>
+      <section>
+        <div class="about-myself-and-skill-list-section">
+          <div class="about-myself-section">
+            <div class="container-for-about-section">
+              <div class="about-section-item">
+                <h1 class="intro">About Myself</h1>
```

Comment:
- [ ] You should only have one `h1` per page. A page can only have one main heading and the rest are sub headings. 
![h](https://github.com/CobbyElsonfx/Portfolio-Website/assets/99656687/a04b92b5-2aad-42e8-b270-e355cc228a4c)